### DISCUSSION TITLE
Avatar Builder Help

### DISCUSSION THREAD

**Discussion by SanginiG** [flyerhq/flutter_chat_ui#492] [Q&A] [answered]
Hi, I used a custom avatar builder, but it seems that i have to fix my avatars size to 40px, this is because flutter_chat_ui-1.6.9/lib/src/widgets/message/message.dart has a SizedBox of width 40. at line no. 194. This creates an issue when messages are grouped and the grouped message bubbles are slightly off from the right side. is their any way to fix this?

**Comment by demchenkoalex** [comment]
Hey, so if the size is restricted and you use the same size, and avatars look good by default - why there will be an issue with bubbles off the right side? Any visuals?

**Reply by SanginiG** [reply, marked as answer]
Hi thanks for responding, the issue got fixed by using my custom avatar inside the bubble builder itself, and the issue only recreated when I had to use small sized avatars instead of the size 40, and when the messages were grouped.
